### ISSUE TITLE
Warning in console when single tab used

### ISSUE THREAD

**Issue by albenik** [mlaursen/react-md#142]
Warning appears in console, when use only single tab:

``` jsx
<Tabs primary scrollable>
    <Tab label="Single tab"><p>Tab content</p></Tab>
</Tabs>
```

`Failed prop type: Invalid prop`children`of type`object`supplied to`Tabs`, expected an array.`

When use multiple tabs, no warnings in console

**Comment by mlaursen** [comment]
I can update the PropTypes to allow a single element. I didn't see a use case for a single tab when setting it up, so I enforced at least 2.

**Comment by albenik** [comment]
@mlaursen I my case tabs represents sections which config loads dynamicaly, so it is possible to recieve only single section in some cases.
Of course I can add some code and don't use tabs if only one section recieved. But it will be nice if I can avoid it.

**Comment by mlaursen** [comment]
This has been fixed for the `v1.0.0` release. I will close this issue once the alpha has been released.
